gslab_scons/_release_tools.py

```diff
@@ -223,29 +220,42 @@ def up_to_date(mode = 'scons', directory = '.', scons_local_path = None):
         # argument is actually a SCons directory.
         # We use the fact that running scons outside of SCons directory
         # produces the message: "No SConstruct file found."
-        if [True for out in output if re.search('No SConstruct file found', out)]:
-            raise ReleaseError('up_to_date(mode = scons) must be run on a '
+        if check_list_for_regex('No SConstruct file found', output):
+            raise ReleaseError('up_to_date(mode = scons) must be run on an '
```

Comment:
it's still a scone over an scone no?

Reply:
I've seen it both ways. Will switch back.

Reply:
Also, who writes SCone?

Reply:
ah I forgot it's pronounced es-scon. Feels so wrong.

![image](https://user-images.githubusercontent.com/20933669/30136592-80591648-9314-11e7-899d-1c545b228481.png)